<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyCoachMainWindow</class>
 <widget class="QMainWindow" name="MyCoachMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Federación de Árbitros de Cádiz</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/iconos/images/icon.png</normaloff>:/iconos/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layoutChronometre">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>FECHA:    </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="dateLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>HORA:     </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="hourLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>10:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Prueba a realizar</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QRadioButton" name="radio60x40Button">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Carrera 6x40 Metros</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio5x40Button">
             <property name="text">
              <string>Carrera 5x40 Metros</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio2000Button">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Carrera 2000 Metros</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioPCButton">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Prueba de campo</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Sesión</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="checkBox6x40">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Carrera 6x40 metros</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox5x40">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Carrera 5x40 metros</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox2000">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Carrera 2000 metros</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxPC">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Prueba de campo</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="cargarSesionButton">
             <property name="text">
              <string>Cargar</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="botoneraLayout">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/iconos/images/timer.png</normaloff>:/iconos/images/timer.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>72</width>
            <height>72</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/iconos/images/stop.png</normaloff>:/iconos/images/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>72</width>
            <height>72</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="raceTable">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="refereeButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/iconos/images/referee.png</normaloff>:/iconos/images/referee.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="reportButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/iconos/images/report.png</normaloff>:/iconos/images/report.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="configButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/iconos/images/settings.png</normaloff>:/iconos/images/settings.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="verBonusButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/images/bonus_view.png</normaloff>:/images/bonus_view.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="dorsalTagsButton">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/iconos/images/dorsales.png</normaloff>:/iconos/images/dorsales.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="bonusButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/iconos/images/bonus.png</normaloff>:/iconos/images/bonus.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Datos de la prueba</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Nº Vueltas</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="vueltasLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Distancia vuelta</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="distVueltaLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Distancia total</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="distTotalLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Nº Corredores</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="corredoresLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Bonificaciones</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="bonusLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
